### DISCUSSION TITLE
Beginner Help: I used invert-paths; are the files still supposed to be listed in the commit message?

### DISCUSSION THREAD

**Discussion by whitemagehealing** [newren/git-filter-repo#491] [Q&A] [answered]
So, I used a command that was similar to the example:

```
git filter-repo --path README.md --path guides/ --path tools/releases --invert-paths
```

It seemed to work okay according to `--analyze` results in `.git`, but if I look at the git history with GitLens  or Git History in VS Code, I expected that the undesired filenames would be stripped out of all the commits messages / digests.  However, the file names still showed that the files were committed. I also understand that the TEXT of the message body would probably stay the same unless I manually hunted for it and changed it.

**So, is this the expected behavior, or did I do something wrong? If this is expected, is there a way to get the files removed from the list of files in the commit list?** 

Sorry, I'm still learning git as well, so I don't even know the correct terminology. Feel free to enlighten me regarding them. If this is a duplicate question, I'm sorry again. It is because I don't know how to phrase what I'm asking about.

**Comment by me-and** [comment, marked as answer]
This is expected behaviour: your files and your commit messages are different things. filter-repo won't change your commit messages unless you explicitly ask it to.
